pkg/clustersql/settings.go

```diff
@@ -0,0 +1,137 @@
+/*
```

Comment:
This and the following file are already in the [`scale` package](https://github.com/cockroachdb/cockroach-operator/blob/master/pkg/scale/timeout.go)